spikeinterface/sorters/utils/misc.py

```diff
@@ -56,3 +64,77 @@ def get_channel_property(self, channel_id, property):
         ind = rec.ids_to_indices([channel_id])
         v = values[ind[0]]
         return v
+
+class NewAPIRecordingExtractor(BaseRecording):
```

Comment:
@samuelgarcia I totally agree with you that the name is ambiguous and a better name is needed. By the way I only chose this name because it was suggested to me by Alessio in our email conversation - so he should be blamed for it! :D

Reply:
Tentatively changed the class names - @alejoe91 let me know what you think!